app/config/config.yml

```diff
@@ -31,12 +31,12 @@ framework:
 
 doctrine:
     dbal:
-        driver: '%database_driver%'
         host: '%database_host%'
         port: '%database_port%'
         dbname: '%database_name%'
         user: '%database_user%'
         password: '%database_password%'
+        driver: pdo_mysql
```

Comment:
Why not keep that parameter?

Reply:
Because it cannot be used in any meaningful way. :smile:

Reply:
Reverted this change for now.